# contracts/interfaces/IERC20Confidential.sol  ->  contracts/interfaces/IERC20Confidential.fsol
# upstream 5138cb8 .. ported 86f1b7c

diff --git a/contracts/interfaces/IERC20Confidential.sol b/contracts/interfaces/IERC20Confidential.fsol
index 48a2341..da0a421 100644
--- a/contracts/interfaces/IERC20Confidential.sol
+++ b/contracts/interfaces/IERC20Confidential.fsol
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.25;
 
 import { euint64, sharedEuint64 } from "@fhenixprotocol/cofhe-contracts/FHE.sol";
-import { IFHERC20 } from "../interfaces/IFHERC20.sol";
+import { IFHERC20 } from "../interfaces/IFHERC20.fsol";
 
 /**
  * @dev Interface for ERC-20 tokens extended with a second, confidential (FHE-encrypted) balance layer.
